src/App.js

```diff
@@ -80,6 +105,22 @@ const App = () => {
                 cartContent={cartContent}
                 isCartVisible={isCartVisible}
                 closeCart={closeCart}
+                wishList={wishList}
+                handleWishList={handleWishList}
+              />
+            }
+          />
+          <Route
+            path='store/games/:gameName'
+            element={
+              <GameDetails
```

Comment:
IMO you should learn about the context API and use it here, all this cart functionality is the perfect use case and atm we are promoting a semi-bad pattern of prop drilling.